src/types/multiModel.js

```diff
@@ -15,14 +13,12 @@ class MultiModel extends BaseType {
         this.options = options || {};
         this.map = this.options.map;
         this.Model = Model;
-        this.requestCache = new RequestCache();
         this._listeners = {};
-        this._models = {};
     }
 
     _instantiateModel (id: ?string): Object {
         const { Model } = this;
-        const model = new Model(this._parent, this._parent, id, this.options.modelOptions);
+        const model = new Model(this._parent, id, this.options.modelOptions);
```

Comment:
@peteigel what are your concerns? Let's say we want to create a new Performer that an Event has. The nice way to do that since we are already deep into the event at this point in the tree would be:

```
const newPerformer = event.get('performers').create();

// Do stuff...

events.add(newPerformer);
```